include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -397,6 +397,10 @@ class CloudEnv : public Env {
   virtual Status DeleteDbid(const std::string& bucket_prefix,
                             const std::string& dbid) = 0;
 
+  Logger* GetLogger() const { return info_log_.get(); }
```

Comment:
Isn't it better to have 

std::shared_ptr<Logger> GetLogger()

otherwise, how do you know when the caller who retrieved the pointer to the logger is done with it? 

@igorcanadi any opinions?

Reply:
I agree that using shared_ptr would be better, but reference counting is not cheap, especially when we do it for every `Log`, even the ones with `DEBUG_LEVEL`. This is probably fine, as the caller can ensure that `CloudEnv` survives its usage of the logger.